osu.Server.Spectator/Hubs/Metadata/MetadataHub.cs

```diff
@@ -129,7 +129,14 @@ public async Task UpdateStatus(UserStatus? status)
                 if (usage.Item.UserStatus != status)
                 {
                     usage.Item.UserStatus = status;
-                    await broadcastUserPresenceUpdate(usage.Item.UserId, usage.Item.ToUserPresence());
+
+                    if (status == UserStatus.Offline)
```

Comment:
Any reason why this is a local check that isn't using the helper method? (`shouldBroadcastPresentToOtherUsers()` I mean)

Reply:
It's actually a different condition – this case we explicitly *want* to sent the null status just once when a user transitions to "appear offline". See comment just below, hopefully.